src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorPicker.xaml

```diff
@@ -0,0 +1,92 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:local="using:Avalonia.Controls"
+        xmlns:converters="using:Avalonia.Controls.Converters"
+        x:CompileBindings="False">
```

Comment:
Note that I cannot use compiled bindings here as it throws errors. I didn't investigate this much.

Reply:
TargetType should be set on ControlTemplate.
And DataType should be set on DataTemplate.
In some cases, compiler can find out about type from the context, but not sure about current situation. At least Rider is confused without these types, but Rider has its own analyzers.

It seems there is another problem with Avalonia Xaml compiler:
```
  A binding with a TemplatedParent RelativeSource has to be in a ControlTemplate. Line 120, position 23. 
```

Reply:
Yea, I know about setting the types but there were additional errors like the one you mentioned so I didn't bother looking at this further.

Reply:
I think I found the issue. Will try to fix it later.

Reply:
I added several x:Type declarations just in case you are able to fix the compiler. Otherwise, I'm also fine with leaving it as-is.

Reply:
This was fixed with #8466, Thanks!